heapster-saw/examples/_CoqProject

```diff
@@ -2,37 +2,40 @@
 -Q ../../saw-core-coq/coq/handwritten/CryptolToCoq   CryptolToCoq
 -Q . Examples
 
+# FIXME: Uncomment _proofs files when they're updated with the latest automation
```

Comment:
Worth filing an issue about this as a reminder.

Reply:
Good suggestion! Done in #1781 and #1783